web-marketplace/src/components/organisms/CreateSellOrderForm/CreateSellOrderForm.constants.ts

```diff
@@ -0,0 +1,5 @@
+export const BASIC_INFO_NAME_LABEL = 'Project name';
+export const BASIC_INFO_NAME_DESCRIPTION =
+  'This is the name of the farm, ranch, property, or conservation project.';
+export const BASIC_INFO_NAME_PLACEHOLDER = 'i.e. Sunnybrook Farms';
+export const BASIC_INFO_SIZE_LABEL = 'Size in hectares or acres';
```

Comment:
why is this here? probably some leftovers from copying/pasting?

Reply:
You're right, will clean up that.